pkg/controller/dgraph/models/group.go

```diff
@@ -0,0 +1,55 @@
+package models
+
+import (
+	"time"
+
+	groups_v1 "github.com/vmware/purser/pkg/apis/groups/v1"
+	"github.com/vmware/purser/pkg/controller/dgraph"
+)
+
+// Dgraph Model Constants
+const (
+	IsGroupCRD = "isGroupCRD"
+)
+
+// GroupCRD schema in dgraph
+type GroupCRD struct {
+	dgraph.ID
+	IsGroupCRD bool      `json:"isGroupCRD,omitempty"`
+	Name       string    `json:"name,omitempty"`
+	StartTime  time.Time `json:"startTime,omitempty"`
```

Comment:
StartTime should be string in dgraph, see other k8s resources. While storing it we need the time in RFC3339 format. Same for subscriber.